feat(content): Korath Probes, Ra'at Adjustment (#7562)
endless-sky/endless-sky: data/korath/korath ships.txt

	description "	Fighters do not come equipped with a hyperdrive. You cannot carry a fighter unless you have a ship in your fleet with a fighter bay."

ship "Kas'ik Tek 11"
	sprite "ship/kas-ik tek 11"
	thumbnail "thumbnail/kas-ik tek 11"
	attributes
		category "Drone"
		"cost" 210000
		"shields" 1400
		"hull" 800
		"automation" 1
		"mass" 21

> 		"heat dissipation" 1
> 		"drag" 0.6
> 		"outfit space" 48
> 		"cargo space" 4

Reviewer: ```suggestion
		"drag" 0.6
		"heat dissipation" 1
		"cargo space" 4
		"outfit space" 48
```

	description "	Fighters do not come equipped with a hyperdrive. You cannot carry a fighter unless you have a ship in your fleet with a fighter bay."

> 

Reviewer 2: ```suggestion
```

ship "Kas'ik Tek 7"

> 	sprite "ship/kas-ik tek 11"
> 	thumbnail "thumbnail/kas-ik tek 11"

Reviewer 2: Either you missed these ones or I can understand why you didn't want to.  If the latter though, your copyright needs to go back to 11

```suggestion
	sprite "ship/kas-ik tek 7"
	thumbnail "thumbnail/kas-ik tek 7"
```

Author: Oops

	attributes
		category "Drone"
		"cost" 210000
		"shields" 1400
		"hull" 800
		"automation" 1
		"mass" 21
		"drag" 0.6
		"heat dissipation" 1
		"cargo space" 4
		"outfit space" 48
		"engine capacity" 18
		"atmosphere scan" 100
		"outfit scan power" 18
		"outfit scan speed" 2
		"cargo scan power" 6
		"cargo scan speed" 2
		"tactical scan power" 30
		"asteroid scan power" 20
		weapon
			"blast radius" 6
			"shield damage" 40
			"hull damage" 30
			"hit force" 80
	outfits
		"Generator (Candle Class)"
		"Small Heat Shunt"
		"Engine (Meteor Class)"
	engine 0 22.5 
	engine -8.5 25 
	engine 8.5 25
	explode "tiny explosion" 9
	explode "small explosion" 6

> 	description `These small probes are deployed by larger Korath Exile ships to retrieve deactivated microbot drones spread across Korath occupied space. The recovered microbots provide valuable data scans and can then be recycled for use afterwards.`
> 	description `	Drones do not come equipped with a hyperdrive. You cannot carry a drone unless you have a ship in your fleet with a drone bay.`

Reviewer 2: All right, now here's the important part of my review :-)

```suggestion
	description `These small probes are deployed by larger Korath Exile ships to scan for intruders, evaluate foreign vessels, and analyse system bodies. Secondarily, they retrieve small debris and deactivated microbots spread across Korath space which have fallen off their ships, been fired from shunt-strikes, or even been deliberately deployed. Although the data observed and logged by an individual microbot is extremely limited, processing them in bulk yields a valuable gestalt of information. More importantly, the microbots can be recycled for use afterwards.`
	description `	Drones do not come equipped with a hyperdrive. You cannot carry a drone unless you have a ship in your fleet with a drone bay.`
```

> 	description `These small probes are deployed by larger Korath Exile ships to scan for intruders, evaluate foreign vessels, and analyze system bodies. Secondarily, they retrieve small debris and deactivated microbots spread across Korath space which have fallen off their ships, been fired from shunt-strikes, or even been deliberately deployed. Although the data observed and logged by an individual microbot is extremely limited, processing them in bulk yields a valuable gestalt of information. More importantly, the microbots can be recycled for use afterwards.`

Reviewer 3: ```suggestion
	description `These small probes are primarily deployed by larger Korath Exile ships to scan for intruders, evaluate foreign vessels, and analyze system bodies. Their secondary function is to retrieve small debris and deactivated microbots spread across Korath space which have fallen off their ships, been fired from Shunt-Strikes, or were even deliberately deployed. Although the data observed and logged by an individual microbot is extremely limited, processing them in bulk yields a valuable gestalt of information. More importantly, the microbots can be recycled for use afterwards.`
```

> 

Reviewer 2: Pattern in this file is two spaces between ships, three between factions
```suggestion
```

	description "The largest of Korath production vessels are derived from a common and ancient chassis. This many-lobed ship has been adapted to carry fighters into large-scale raids, or mining craft into asteroid belts. Its radial arrangements of hangars allows fighters a maximum variety of safe approach patterns in combat."

> 

Reviewer 2: ```suggestion
```
